src/login/LoginPage.jsx

```diff
@@ -247,6 +344,14 @@ LoginPage.defaultProps = {
   shouldBackupState: false,
   showResetPasswordSuccessBanner: false,
   submitState: DEFAULT_STATE,
+  thirdPartyAuthApiStatus: 'pending',
```

Comment:
Lets use the constant `PENDING_STATE`.

Reply:
this is addressed I have used the constant